Makefile

```diff
@@ -82,7 +86,10 @@ example-kubernetes: container
 example-containers: container
 	examples/containers/run.sh
 
-examples: clean container example-kubernetes example-containers
+example-multi: container
+	examples/containers/multi/run.sh
```

Comment:
Should we add a variant for `container-oss`? Otherwise people outside Couchbase can't run the example.

Reply:
Good catch, yeah we should. It might even be better to not link them in this fashion to allow us to do either first or pull the images.

Reply:
Should this also apply to example-containers, example-kubernetes, and example-native (later)?

Reply:
Yes